gr-fec/python/fec/LDPC/Generate_LDPC_matrix_functions.py

```diff
@@ -584,7 +584,7 @@ def get_full_rank_H_matrix(H, verbose=False):
 
   return newH
 
-def get_best_matrix(H, numIterations=100, verbose=False):
+def get_best_matrix(H, numIterations=100, verbose=0):
```

Comment:
I think `verbose=False` should be the preferred type here. It just makes it more obvious that this is a simple switch.

Reply:
The rest of code has different levels of verbose (0, 1 and 2), hence why I changed this one as it was the only place that used True or False. Should the two verbose levels be merged then?

Reply:
I wasn't aware of the 3 verbosity levels. In this case, `int` is the appropriate type.